lib/android/app/src/main/java/com/reactnativenavigation/views/stack/topbar/titlebar/TitleAndButtonsContainer.kt

```diff
@@ -14,64 +12,53 @@ import com.reactnativenavigation.options.parsers.TypefaceLoader
 import com.reactnativenavigation.utils.CompatUtils
 import com.reactnativenavigation.utils.UiUtils
 import com.reactnativenavigation.utils.ViewUtils
-import com.reactnativenavigation.utils.logd
 
 const val DEFAULT_LEFT_MARGIN = 16
 
-class MainToolBar(context: Context) : RelativeLayout(context) {
+
+class TitleAndButtonsContainer(context: Context) : RelativeLayout(context) {
+
+    private val defaultMargin = UiUtils.dpToPx(context, DEFAULT_LEFT_MARGIN)
```

Comment:
we do have this in the buttons toolbars,
![Screen Shot 2021-04-27 at 20 01 00](https://user-images.githubusercontent.com/7227793/116282642-b09c5a80-a793-11eb-80ac-d07a9a610efb.png)

 `defaultMargin` is used to add padding at the left side of the title component, now we have 3 components, I have to do it manually because the title is not inside a `Toolbar`